RIOT-OS/RIOT
# congure_reno: initial import of TCP Reno congestion control

### Contribution description
This provides an implementation for CongURE of the congestion control mechanism of classic TCP congestion control (TCP Reno) as specified in [RFC 5681](https://tools.ietf.org/html/rfc5681).




### Testing procedure
Tests are provided for which [`riotctrl`](https://github.com/RIOT-OS/riotctrl/) (at least v0.2.2) is required for them to run. If installed,

```
make -j -C tests/congure_reno flash test
```

should succeed.



### Issues/PRs references
Requires ~~#15951~~ (merged), ~~#16111, and #16119~~ (merged). When merged, I will adapt the test framework to use the turo framework introduced in #15950, which was the original idea to begin with (which is why it is already printing JSON strings).

![PR dependency graph](https://page.mi.fu-berlin.de/mlenders/congure.svg)
